.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Chemprop - CPU",
+    "build": {
+        "context": "..",
+        "dockerfile": "../Dockerfile"
+    },
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
+    },
+    "postCreateCommand": "pip install -e .",
+    "extensions": [
+        "ms-python.python",
```

Comment:
```suggestion
        "ms-python.python",
        "ms-toolsai.jupyter",
```

Reply:
We probably also need to install `ipykernel` via `pip install -e ".[notebooks]"`